### DISCUSSION TITLE
No wallpaper in Hyrpland

### DISCUSSION THREAD

**Discussion by DkmenchaR** [JaKooLit/Hyprland-Dots#244] [Q&A] [answered]
Hi guys, I've recently reinstalled arch in my desktop, and for some reason my background is always black, no matter how many times i change wallpaper for force with only one in UserConfigs
I don't know if I'm missing something, but previous JaKooLit Hyprland dots were fine... I can select them, the animation appears, but no background.. everything else works normally

**Comment by DkmenchaR** [comment, marked as answer]
Nevermind.... Just recompiled swww, systemctl reboot and it fixed!

**Reply by ghost** [reply]
Sir can please explain step by step?
